__tests__/api.test.js

```diff
@@ -29,3 +29,60 @@ describe("3. GET /api/topics", () => {
       });
   });
 });
+
+describe("4. GET /api/articles/:article_id", () => {
+  test("should return  status 200", () => {
+    return request(app).get("/api/articles/1").expect(200);
+  });
+  test("should respond with an article obj with correct properties", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .then(({ body }) => {
+        const objOfCorrectShape = {
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
```

Comment:
If you're making a request for a specific article_id, you want to make sure you're getting the right one back, so you should hardcode the id here :)